### PR TITLE
Update add marble specific site.

### PR DESCRIPTION
**Pull request by jonhartzler**
Added tests back in 
Fixed SEO display

Added a test library that does snapshot comparison.   
https://reactjs.org/docs/test-renderer.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,25 +10,31 @@
...
     "mirador": "^3.0.0",
     "mirador-image-tools": "^0.10.0",
-    "typy": "^3.1.0",
+    "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-helmet": "^6.1.0"
+    "react-helmet": "^6.1.0",
+    "typy": "^3.1.0",
+    "@loadable/component": "^5.14.1"
```
Comment (jonhartzler) [inline package.json:24]: Just wanted to call this line out @danthewolfe.  I use this library to not load mirador in test.

**Review by danthewolfe — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (danthewolfe) [reply package.json:24]: 👍  I could not get this to work right in MARBLE when I tried using it before. I am glad it works in this scenario.

Comment (jonhartzler) [reply package.json:24]: nm I was mistaken this does not work

Comment (jonhartzler) [reply package.json]: replaced with a mock which does work.

Comment (danthewolfe) [reply package.json]: 👍

**Review by danthewolfe — APPROVED** [review]

**Pull request merged**